qiskit_ibm_runtime/utils/json.py

```diff
@@ -270,10 +270,9 @@ def default(self, obj: Any) -> Any:  # pylint: disable=arguments-differ
             return {"__type__": "BitArray", "__value__": out_val}
         if isinstance(obj, DataBin):
             out_val = {
-                "field_names": obj._FIELDS,
-                "field_types": [str(field_type) for field_type in obj._FIELD_TYPES],
```

Comment:
@kt474 I removed `field_types` because it is not necessary for DataBin for Qiskit 1.1. Could you confirm whether it works fine with the server side? If necessary, I will revert this line.